data/ecosystems/d/dfinity-education.toml

```diff
@@ -0,0 +1,24 @@
+title = "Internet Computer Education"
```

Comment:
All these repos might already be under ICP main ecosystem file. Example: https://github.com/electric-capital/crypto-ecosystems/blob/bbdf537935ae1d48227c96a995359f581ab51fd2/data/ecosystems/i/internet-computer.toml#L2773